tokio/src/task/blocking.rs

```diff
@@ -6,18 +6,23 @@ cfg_rt_multi_thread! {
     ///
     /// In general, issuing a blocking call or performing a lot of compute in a
     /// future without yielding is not okay, as it may prevent the executor from
-    /// driving other futures forward.  This function runs the closure on the
-    /// current thread by having the thread temporarily cease from being a core
-    /// thread, and turns it into a blocking thread. See the [CPU-bound tasks
-    /// and blocking code][blocking] section for more information.
-    ///
-    /// Although this function avoids starving other independently spawned
-    /// tasks, any other code running concurrently in the same task will be
-    /// suspended during the call to `block_in_place`. This can happen e.g. when
-    /// using the [`join!`] macro. To avoid this issue, use [`spawn_blocking`]
-    /// instead.
-    ///
-    /// Note that this function can only be used when using the `multi_thread` runtime.
+    /// driving other tasks forward. By using this function, the executor
```

Comment:
"Calling this function informs the executor that the currently executing task is about to block the thread"

maybe? TOLI